cmd/oras/internal/option/remote.go

```diff
@@ -50,6 +50,8 @@ import (
 type Remote struct {
 	DistributionSpec
 	CACertFilePath    string
+	CertFilePath      string
+	KeyFilePath       string
```

Comment:
Should users pass the key and cert file to ORAS from local filesystem with `--CertFilePath` and `KeyFilePath`?

Reply:
@FeynmanZhou similar to the `--ca-file` option,  its the location of the file(s) containing the client certificate and associated private key

Reply:
> Should users pass the cert and key file to ORAS from local filesystem with --CertFilePath and KeyFilePath?

User should pass `--cert-file <CertFilePath>` and `--key-file <KeyFilePath>` to ORAS